common/src/main/java/com/revenuecat/purchases/common/HTTPClient.kt

```diff
@@ -158,6 +158,7 @@ class HTTPClient(
             "X-Version" to Config.frameworkVersion,
             "X-Client-Locale" to appConfig.languageTag,
             "X-Client-Version" to appConfig.versionName,
+            "X-Client-Package-Name" to appConfig.packageName,
```

Comment:
I wasn't sure what name to use for the header... In the end I picked the `Client` prefix since it's what we have in iOS (`X-Client-Bundle-ID`), followed by `Package-Name` since it's the name of this in Android.

Reply:
I know it's not called "Bundle-ID" in Android, but I wonder if the server would prefer using the same key for this as in iOS?

Reply:
let's re-use the same key so it's easier for us when trying to extract the data